### DISCUSSION TITLE
Documentation Electric Modell, Transformer

### DISCUSSION THREAD

**Discussion by scud-soptim** [PowerGridModel/.github#25] [Q&A] [answered]
I think there is a small mistake in the formula to calculate z_base. You wrote: z_base = 1/y_base = S_base/(u2,rated*u2,rated). However, it should be: z_base = 1/y_base; y_base = S_base/(u2,rated*u2,rated).

**Comment by TonyXiang8787** [comment, marked as answer]
Hi @sudo-ac, you are correct. This is indeed a fault. Could you please submit a PR to fix this?

**Comment by scud-soptim** [comment]
I do my best ...
